### PR TITLE
AG-9515: Move interval options out of `axes.ticks`

### PR DESCRIPTION
**Pull request by iMoses**

### CODE REVIEW

File: packages/ag-charts-website/src/content/docs/axes-ticks/_examples/axis-tick-interval/main.ts [hunk 1]
```diff
@@ -28,21 +28,19 @@ const options: AgCartesianChartOptions = {
...
+            interval: 20,
         },
     ],
 };
 
 const chart = AgCharts.create(options);
 
 function setTickInterval(interval: number) {
-    options.axes![1].tick = { interval };
+    Object.assign(options.axes![1], { interval });
```
Comment (alantreadway) [inline packages/ag-charts-website/src/content/docs/axes-ticks/_examples/axis-tick-interval/main.ts:39]: Nit: Maybe `options.axes![1].interval = interval`?

File: packages/ag-charts-website/src/content/docs/axes-ticks/_examples/axis-tick-interval/main.ts [hunk 2]
```diff
@@ -28,21 +28,19 @@ const options: AgCartesianChartOptions = {
...
     chart.update(options);
 }
 
 function resetInterval() {
-    options.axes![1].tick = {};
+    Object.assign(options.axes![1], { interval: undefined });
```
Comment (alantreadway) [inline packages/ag-charts-website/src/content/docs/axes-ticks/_examples/axis-tick-interval/main.ts:44]: Nit: Maybe `options.axes![1].interval = undefined`?

File: packages/ag-charts-website/src/content/docs/axes-ticks/_examples/axis-tick-min-max-spacing/main.ts [hunk 3]
```diff
@@ -28,21 +28,19 @@ const options: AgCartesianChartOptions = {
...
+            interval: 20,
         },
     ],
 };
 
 const chart = AgCharts.create(options);
 
 function setMinMaxSpacing(minSpacing: number, maxSpacing: number) {
-    options.axes![1].tick = { minSpacing, maxSpacing };
+    Object.assign(options.axes![1], { minSpacing, maxSpacing });
```
Comment (alantreadway) [inline packages/ag-charts-website/src/content/docs/axes-ticks/_examples/axis-tick-min-max-spacing/main.ts:39]: Ditto

File: packages/ag-charts-website/src/content/docs/axes-ticks/_examples/axis-tick-min-max-spacing/main.ts [hunk 4]
```diff
@@ -28,21 +28,19 @@ const options: AgCartesianChartOptions = {
...
     chart.update(options);
 }
 
 function reset() {
-    options.axes![1].tick = {};
+    Object.assign(options.axes![1], { minSpacing: undefined, maxSpacing: undefined });
```
Comment (alantreadway) [inline packages/ag-charts-website/src/content/docs/axes-ticks/_examples/axis-tick-min-max-spacing/main.ts:44]: Ditto

**Review by alantreadway — APPROVED** [review]
LGTM, some minor nits, but let's follow up distinctly.

File: packages/ag-charts-website/src/content/docs/axes-ticks/_examples/axis-tick-interval/main.ts (hunk 1 above)
Comment (iMoses) [reply packages/ag-charts-website/src/content/docs/axes-ticks/_examples/axis-tick-interval/main.ts:39]: The problem with all these cases was it required a specific type to pass TS checks, so the alternative is actually:
`(options.axes![1] as AgContinuousAxisOptions).interval = interval`

**Pull request merged**